tests/bluetooth/tester/src/btp_micp.c

```diff
@@ -0,0 +1,534 @@
+	}
+#endif /* CONFIG_BT_MICP_MIC_CTLR_AICS */
+};
+
+static uint8_t micp_supported_commands(const void *cmd, uint16_t cmd_len,
+				       void *rsp, uint16_t *rsp_len)
+{
+	struct btp_micp_read_supported_commands_rp *rp = rsp;
+
+	/* octet 0 */
+	tester_set_bit(rp->data, BTP_MICP_READ_SUPPORTED_COMMANDS);
+
+	*rsp_len = sizeof(*rp) + 1;
+
+	return BTP_STATUS_SUCCESS;
+}
+
+static uint8_t micp_discover(const void *cmd, uint16_t cmd_len,
+			     void *rsp, uint16_t *rsp_len)
+{
+	const struct btp_micp_discover_cmd *cp = cmd;
+	struct bt_conn *conn;
+	int err;
+
+	conn = bt_conn_lookup_addr_le(BT_ID_DEFAULT, &cp->address);
+
+	if (!conn) {
+		LOG_ERR("Unknown connection");
+		return BTP_STATUS_FAILED;
+	}
+
+	err = bt_micp_mic_ctlr_discover(conn, &mic_ctlr);
+
+	if (err != 0) {
+		LOG_DBG("Fail: %d", err);
+		return BTP_STATUS_FAILED;
+	}
+
+	return BTP_STATUS_SUCCESS;
+}
+
+static uint8_t micp_set_gain(const void *cmd, uint16_t cmd_len,
+			     void *rsp, uint16_t *rsp_len)
+{
+	const struct btp_micp_set_gain_cmd *cp = cmd;
+
+	LOG_DBG("MICP set gain %d", cp->gain);
+
+	for (int i = 0; i < micp_included.aics_cnt; i++) {
+		if (bt_aics_gain_set(micp_included.aics[i], cp->gain) != 0) {
+			return BTP_STATUS_FAILED;
+		}
+	}
+
+	return BTP_STATUS_SUCCESS;
+}
+
+static uint8_t micp_aics_unmute(const void *cmd, uint16_t cmd_len,
+				void *rsp, uint16_t *rsp_len)
+{
+	LOG_DBG("Unmute");
+
+	for (int i = 0; i < micp_included.aics_cnt; i++) {
+		if (bt_aics_unmute(micp_included.aics[i]) != 0) {
+			return BTP_STATUS_FAILED;
+		}
+	}
+
+	return BTP_STATUS_SUCCESS;
+}
+
+static uint8_t micp_aics_mute(const void *cmd, uint16_t cmd_len,
+			      void *rsp, uint16_t *rsp_len)
+{
+	LOG_DBG("Mute");
+
+	for (int i = 0; i < micp_included.aics_cnt; i++) {
+		if (bt_aics_mute(micp_included.aics[i]) != 0) {
+			return BTP_STATUS_FAILED;
+		}
+	}
+
+	return BTP_STATUS_SUCCESS;
+}
+
+static uint8_t micp_aics_state(const void *cmd, uint16_t cmd_len,
+			       void *rsp, uint16_t *rsp_len)
+{
+	LOG_DBG("AICS State");
+
+	for (int i = 0; i < micp_included.aics_cnt; i++) {
+		if (bt_aics_state_get(micp_included.aics[i]) != 0) {
+			return BTP_STATUS_FAILED;
+		}
+	}
+
+	return BTP_STATUS_SUCCESS;
+}
+
+static uint8_t micp_aics_type(const void *cmd, uint16_t cmd_len,
+			      void *rsp, uint16_t *rsp_len)
+{
+	LOG_DBG("AICS Type");
+
+	for (int i = 0; i < micp_included.aics_cnt; i++) {
+		if (bt_aics_type_get(micp_included.aics[i]) != 0) {
+			return BTP_STATUS_FAILED;
+		}
+	}
+
+	return BTP_STATUS_SUCCESS;
+}
+
+static uint8_t micp_aics_status(const void *cmd, uint16_t cmd_len,
+				void *rsp, uint16_t *rsp_len)
+{
+	LOG_DBG("AICS Status");
+
+	for (int i = 0; i < micp_included.aics_cnt; i++) {
+		if (bt_aics_status_get(micp_included.aics[i]) != 0) {
+			return BTP_STATUS_FAILED;
+		}
+	}
+
+	return BTP_STATUS_SUCCESS;
+}
+
+static uint8_t micp_mute_read(const void *cmd, uint16_t cmd_len,
+			      void *rsp, uint16_t *rsp_len)
```

Comment:
Some commands use a verb `read`, `set`, etc. but some are just `micp_aics_status`. Suggest to be consistent and use verbs for all functions